tests/server/bundle_manager_test.py

```diff
@@ -3,16 +3,11 @@
 
 from codalab.objects.metadata_spec import MetadataSpec
 from codalab.server.bundle_manager import BundleManager
+from codalab.worker.bundle_state import RunResources
 from codalab.bundles import RunBundle
 
 
 class BundleManagerTest(unittest.TestCase):
-    def setUp(self):
-        self.manager = BundleManager()
-        self.manager._default_request_cpus = 1
-        self.manager._default_request_gpus = 0
-        self.manager._default_request_queue = None
-
```

Comment:
No need to set up a full on `BundleManager` object anymore since we're just testing the static worker sorting method.